_pages/technical-details.markdown

```diff
@@ -62,9 +64,45 @@ of a Semantic Web-based information system as smooth as possible.
 There is also a [library](https://github.com/ledsoft/jopa-spring-transaction) for integrating JOPA with the declarative
 transaction management of Spring (think `@Transactional`) that is used in TermIt.
 
-#### JB4JSON-LD
+##### JB4JSON-LD
 [JB4JSON-LD](https://github.com/kbss-cvut/jb4jsonld) (Java Binding for JSON-LD) is a library for seamless marshalling and unmarshalling of POJOs into and from JSON-LD.
 It integrates with Jackson, so the set-up for an application is a matter of just a couple of lines of code.
 
 
-# Data Model
+## Data Model
+
+Vocabularies modelled in TermIt are using two specific data models -- Data description ontology and TermIt ontology, both developed by Knowledge-based and Software Systems Group (KBSS).
+
+### Data description ontology
+
+Basic ontology used for description of any data is called Data description ontology and is described on [this page](http://onto.fel.cvut.cz/ontologies/slovnik/agendovy/popis-dat/current/index-en.html). It is based on the Unified Foundational Ontology (UFO).
+
+The current version of Data description ontology can be found [here](https://onto.fel.cvut.cz/ontologies/page/slovnik/agendovy/popis-dat/model/verze/1.0.1) in turtle or RDF/XML.
+
+Following sections describes basic classes defined in the model.
+
+#### Vocabulary
+Vocabulary represents an ontology, consists of glossary and model.
+
+#### Glossary
+Glossary is a list of all terms present in vocabulary, using only skos concepts.
+
+#### Model
+Model brings ontological relations to the terms defined in the glossary, using mainly UFO.
+
+#### Term
+Term is any concept used in the vocabulary.
+
+#### Attribute
+Attribute is a special type of term, representing a dependent property. It may represent data attributes, reference attributes or statistical attributes.
```

Comment:
Will the reader understand what 'dependent property' is?

Reply:
I am not sure. Do you have better idea how to describe that? I spent over ten minutes with this :)

Reply:
Attribute types are terms (e.g. `ship length`, `person name`). An attribute belongs to an object and cannot exist without it.